readme.md

```diff
@@ -1,31 +1,63 @@
-# spn
+# Starport Network
 
-**spn** is a blockchain application built using Cosmos SDK and Tendermint and generated with [Starport](https://github.com/tendermint/starport).
+Starport Network (SPN) is a network for launching [Cosmos](https://cosmos.network) blockchains by matching coordinators with validators. This repository contains an SPN node built with Cosmos SDK and generated with [Starport](https://github.com/tendermint/starport).
```

Comment:
I would put a link for the user to have more information about being a Validator

There is this one https://hub.cosmos.network/master/validators/validator-faq.html but it's related to Cosmos Hub, so I don't know if it makes sense to provide it

Reply:
A "validator" is a Tendermint concept, so we might as well link to the docs. Might not be the most gentle introduction, but it's technically correct (and should be good enough until we have a better article).

https://docs.tendermint.com/master/nodes/validators.html

Reply:
> A "validator" is a Tendermint concept

Depends, you have the Tendermint validator, which is a node that proposes and validates blocks, and the Staking module validator which represents the account with delegated tokens.
With Starport Network want the coordinator to find Tendermint validators to make the chain working, but at the same time, the `AddValidator` proposal is related to the `staking` module so you would need some notion about this as well to understand staking and delegation.

But well, it's not relevant in the readme. That must be described in the docs